doc/classes/VoxelEngine.xml

```diff
@@ -42,6 +42,19 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="get_thread_count" qualifiers="const">
+			<return type="int" />
+			<description>
+				Returns the number of threads currently used internally by the [code]ThreadedTaskRunner[/code].
+			</description>
+		</method>
+		<method name="set_thread_count">
+			<return type="void" />
+			<param index="0" name="count" type="int" />
+			<description>
+				Sets the number of threads to be used internally by the [code]ThreadedTaskRunner[/code]. Setting this can cause lagging, and it might take some time until the number of threads actually matches the given value.
```

Comment:
There are ways to have no lag. I just dont clearly remember why `set_thread_count` destroys all the threads. Maybe there isn't a particular reason other than it was used only in the destructor. But maybe for now it can be with lag.